### ISSUE TITLE
Is person-level classifier for AVA trained end-to-end with SlowFast?

### ISSUE THREAD

**Issue by anuar12** [facebookresearch/SlowFast#25]
The paper says: " We extract region-of-interest (RoI) features [6] at the last feature map of res5. We extend each 2D RoI at a frame into a 3D RoI by replicating it along the temporal axis, following [9]. We compute RoI features by mRoIAlign [10] spatially, and global average pooling temporally. The RoI features are then max-pooled and fed to a per-class, sigmoid-based classifier for multi-label prediction."

As far as I understand there is a separate classifier that operates on the extracted features from the SlowFast. So this person-level action classifier is a different neural net that takes the features of each person (i.e. detections=>ROI-Pool from SlowFast=>Pool=>ClassifierNN) as input and predicts sigmoid-ed classes, right?
Therefore SlowFast network is not actually trained on person-level labels, it is trained on scene-level labels. And person-level classifier is trained separately?

**Comment by takatosp1** [comment]
Thanks for asking!
SlowFast is trained on person-level labels, and everything is trained end-to-end. Only one classifier is used: slowfast->RoI->projection
